cmake/vs_version/vs_version.cmake

```diff
@@ -0,0 +1,88 @@
+# Copyright (C) 2020 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+#
+
+macro(ie_parse_ci_build_number)
+    if(CI_BUILD_NUMBER MATCHES "^([0-9]+)\.([0-9]+)\.([0-9]+)\-.*")
+        set(IE_VERSION_MAJOR ${CMAKE_MATCH_1})
+        set(IE_VERSION_MINOR ${CMAKE_MATCH_2})
+        set(IE_VERSION_PATCH ${CMAKE_MATCH_3})
+        set(IE_VS_VER_HAS_WELL_DEFINED_VERSION 1)
+    else()
+        set(IE_VS_VER_HAS_WELL_DEFINED_VERSION 0)
+    endif()
+endmacro()
+
+ie_parse_ci_build_number()
+
+if(IE_VS_VER_HAS_WELL_DEFINED_VERSION)
+    set(IE_VS_VER_FILEVERSION_QUAD "${IE_VERSION_MAJOR},${IE_VERSION_MINOR},${IE_VERSION_PATCH},0")
+    set(IE_VS_VER_PRODUCTVERSION_QUAD "${IE_VERSION_MAJOR},${IE_VERSION_MINOR},${IE_VERSION_PATCH},0")
+    set(IE_VS_VER_FILEVERSION_STR "${IE_VERSION_MAJOR}.${IE_VERSION_MINOR}.${IE_VERSION_PATCH}.0")
+endif()
+
+set(IE_VS_VER_PRODUCTVERSION_STR "${CI_BUILD_NUMBER}")
+set(IE_VS_VER_PRODUCTNAME_STR "OpenVINO toolkit")
+set(IE_VS_VER_COPYRIGHT_STR "Copyright (C) 2018-2020, Intel Corporation")
+set(IE_VS_VER_COMMENTS_STR "https://docs.openvinotoolkit.org/")
+
+#
+# ie_add_vs_version_file(NAME <name>
+#                        FILEDESCRIPTION <file description>
+#                        OUTPUT_FILE <output file>
```

Comment:
OUTPUT_FILE is not configured in parse arguments

Reply:
removed this option as unused